src/form_components/repeating_group.coffee

```diff
@@ -57,8 +57,13 @@ FormRenderer.Models.ResponseFieldRepeatingGroup = FormRenderer.Models.BaseFormCo
     @entries.length < @maxEntries()
 
 FormRenderer.Models.ResponseFieldRepeatingGroupEntry = Backbone.Model.extend
+  field_type: 'repeating_group_entry'
```

Comment:
Does screendoor ever see this field type?

Reply:
I don't believe so, but I wanted to make sure we had a descriptive field_type on the model instead of `nil` since we check it everywhere, and it seemed wrong to have a one-off field without a field_type. I believe it can be removed though.